packages/project-access/package.json

```diff
@@ -46,6 +46,7 @@
         "@types/mem-fs-editor": "7.0.1",
         "@types/semver": "7.5.2",
         "@ui5/manifest": "1.61.0",
+        "json-parse-even-better-errors": "3.0.2",
```

Comment:
Shouldn't this be declared in `dependencies`?
Additionally, `@types/json-parse-better-errors` can be added in `devDependencies`